src/eel2_adapter.cpp

```diff
@@ -50,34 +50,35 @@ void EEL2Adapter::init(const std::string &script) {
 
   if (mScript->sample.empty()) {
     Print("ERROR: DynGen sample code is missing");
-    return;
+    return false;
   }
 
   if (!mScript->init.empty()) {
     mInitCode = NSEEL_code_compile_ex(mEelState, mScript->init.c_str(), 0, compileFlags);
     if (!mInitCode) {
-      Print("ERROR: DynGen init compile error: %c",  mEelState->last_error_string);
-      return;
+      std::cout << "ERROR: DynGen init compile error: " << mEelState->last_error_string << std::endl;
```

Comment:
Why did you change this from `Print` to `std::cout`?

Reply:
Still running into this problem, with either using `%c`

<img width="1561" height="723" alt="grafik" src="https://github.com/user-attachments/assets/488679f7-1be4-435b-8f50-0f79bd5c88f2" />

or `%s` 

<img width="1606" height="626" alt="grafik" src="https://github.com/user-attachments/assets/1546a7b4-31e2-4c5d-8a1e-824da7b1936c" />